Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,888 @@
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "73":
+    id: "73"
+    taskid: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+    type: regular
+    task:
+      id: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+      version: -1
+      name: Investigate collected data
+      description: |2-
+         To determine if this is a true positive event,  review the operations performed by the access key and the user in the recent time frame.
+        Search for the following items:
+        Investigate the operation performed by the access key and examine the executed operations, by who it was executed, on which resource, and the operation status.
+        Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.
+        Look at any persistence, for example - a new user creation or key, etc.
+        Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.
+        Look at any lateral movement operations. For example, an operation can be = AsumeRole.
+        As an extra validation step, it is recommended to query the user and/or the user’s manager regarding the investigated suspicious activity
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 880,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "80":
+    id: "80"
+    taskid: 3e6e0a3b-c353-46a8-895c-e46d5fcb4efe
+    type: playbook
+    task:
+      id: 3e6e0a3b-c353-46a8-895c-e46d5fcb4efe
+      version: -1
+      name: Enrichment for Verdict
+      description: This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then  returns the information needed to establish the alert's verdict.
+      playbookName: Enrichment for Verdict
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "50"
+    scriptarguments:
+      CloseReason:
+        simple: Resolved - False Positive,Resolved - Duplicate Incident,Resolved - Known Issue
+      Domain:
+        complex:
+          root: alert
+          accessor: domainname
+      FileSHA256:
+        complex:
+          root: alert
+          accessor: initiatorsha256
+      IP:
+        complex:
+          root: alert
+          accessor: hostip
```

Comment:
let's use playbook inputs for host ip

Reply:
In the playbook on XSOAR, Yarden told me to remove this input and change it to incident.hostip

Reply:
OK, if Yarden wants to use less inputs than keep the alert.x keys.